build.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+set -eo pipefail
+
+# Only allow this script to run in package directories
+node -e 'process.exit(require("./package.json").private === true)' || die 'Not in a package directory.'
+
+# dist
+tsc
```

Comment:
Some people might not have typescript installed globally on their machines, let's document the system requirements for some less standard packages required for the scripts including yarn, typescript.

Reply:
Good call we should have this documented somewhere, I am thinking we include it as requirements within all the SF-X related repos? 

But I think that as long as typescript is listed as a devDependency and we're executing the build script as a npm script(a script listed in the package.json) it should work without having typescript installed globally.

Reply:
Okay sounds good, if we think the documentation for this needs to change in the future we can just flag it then.